src/main/java/codeu/controller/RegisterServlet.java

```diff
@@ -67,6 +67,15 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     User user = new User(UUID.randomUUID(), username, hashedPassword, Instant.now());
     userStore.addUser(user);
 
+    String button = request.getParameter("adminbutton");
```

Comment:
Is this just for the sake of the prototype? We probably don't want all users to be able to sign up as admins. Maybe we should start with a hardcoded list of usernames?